linetools/spectra/xspectrum1d.py

```diff
@@ -1421,11 +1426,13 @@ def fit_continuum(self, knots=None, edges=None, wlim=None, dw=10.,
 
         print('Updating continuum.')
         self.co = wrapper.continuum  # this should work with the assignment, even is self.co_is_set is False
-        if 'contpoints' not in self.meta:
-            self.meta['contpoints'] = []
-        self.meta['contpoints'].extend(
-            [tuple(pts) for pts in wrapper.contpoints])
-        self.meta['contpoints'].sort()
+        # Put into meta -- has been causing trouble
+        if False:
```

Comment:
No, I am now explicitly avoiding writing the continuum to the header.
Was causing too much grief.